ons-spark/pyspark-intro/pyspark-intro.ipynb

```diff
@@ -0,0 +1,1550 @@
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    incident_number                      animal_group postcode_district  \\\n",
+       "0  000014-03092018M  Unknown - Heavy Livestock Animal               CR8   \n",
+       "1   000099-01012017                               Dog               BR2   \n",
+       "2   000260-01012017                              Bird               CR0   \n",
+       "3   000375-01012017                               Dog               TW8   \n",
+       "4   000477-01012017                              Deer               HA7   \n",
+       "\n",
+       "   population  \n",
+       "0       32307  \n",
+       "1       44958  \n",
+       "2      153812  \n",
+       "3       20330  \n",
+       "4       36046  "
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "rescue_with_pop = (rescue_with_pop\n",
+    "                   .orderBy(\"incident_number\")\n",
+    "                   .select(\"incident_number\", \"animal_group\", \"postcode_district\", \"population\"))\n",
+    "\n",
+    "rescue_with_pop.limit(5).toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Writing data: file choice\n",
+    "\n",
+    "In this article so far, we have been calling actions to preview the data, bringing back only a handful of rows each time. This is useful when developing and debugging code, but in production pipelines you will want to write the results.\n",
+    "\n",
+    "The format in which the results are written out depends on what you want to do next with the data:\n",
+    "- If the data are intended to be human readable, e.g. as the basis for a presentation, or as a publication on the ONS website, then you will likely want to output the data as a CSV\n",
+    "- If the data are intended to be used as an input to another Spark process, then use parquet or a Hive table.\n",
+    "\n",
+    "There are other use cases, e.g. JSON can be useful if you want the results to analysed with a different programming language, although here we only focus on CSV and parquet. See the article on Output File Choices for more information.\n",
+    "\n",
+    "### Write to a parquet: `.write.parquet()`\n",
+    "\n",
+    "To write out our DataFrame as a parquet file, use [`rescue_with_pop.write.parquet()`](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrameWriter.parquet.html), using the file path as the argument.\n",
+    "\n",
+    "The key difference between writing out data with Spark and writing out data with pandas is that the data will be distributed, which means that multiple files will be created, stored in a parent folder. Spark can read in these parent folders as one DataFrame. There will be one file written out per partition of the DataFrame."
```

Comment:
Could reference the partitions article somewhere here. You'll probably know where to mention it after reviewing that pull request

Reply:
Done